lib/Test/Nginx/Util.pm

```diff
@@ -1430,25 +1438,7 @@ sub run_test ($) {
 
                 if ($UseHup) {
                     if ($first_time) {
-                        if ($Verbose) {
-                            warn "sending QUIT signal to $pid\n";
-                        }
-
-                        if (kill(SIGQUIT, $pid) == 0) { # send quit signal
-                            #warn("$name - Failed to send quit signal to the nginx process with PID $pid");
-                        }
-
-                        my $max_i = 15;
-                        for (my $i = 1; $i <= $max_i; $i++) {
-                            last unless is_running($pid);
-
-                            sleep $TestNginxSleep;
-                            next if $i < $max_i;
-
-                            warn "WARNING: $name - killing nginx $pid with force...";
-                            kill(SIGKILL, $pid);
-                            waitpid($pid, 0);
-                        }
+                        kill_process($pid, 1, $name);
```

Comment:
we should `kill_process($pid, 0, $name);` to kill nginx?
For now, it won't kill force.